### PR TITLE
Fix auto verify

### PR DESCRIPTION
**Pull request by Natty-Boh**
also makes rank job update team roles

### CODE REVIEW

File: src/handler/job/monitorRanks.ts [hunk 1]
```diff
@@ -1,17 +1,36 @@
...
+} from "../../service/verifyService";
+import { Color } from "../../service/util/constants";
 
 const logger = makeLogger(module);
 
 @EventHandler()
 @ScheduledJob()
 export class MonitorRanks {
-    @Cron("*/10 * * * *")
+    @Cron("* * * * *")
```
Comment (TotallyNot) [inline src/handler/job/monitorRanks.ts]: Is this intentional?

File: src/handler/job/monitorRanks.ts [hunk 2]
```diff
@@ -20,4 +39,52 @@ export class MonitorRanks {
...
+                    Color.RED
+                );
+            } else {
+                logger.error(e);
+            }
+        }
+    }
+
+    @DiscordEvent("guildMemberRemove")
+    async memberLeft(member: GuildMember | PartialGuildMember) {
```
Comment (TotallyNot) [inline src/handler/job/monitorRanks.ts:70]: will this cause issues if this doesn't complete successfully for some reason?

File: src/handler/job/monitorRanks.ts [hunk 3]
```diff
@@ -20,4 +39,52 @@ export class MonitorRanks {
...
+        logger.info(`${member.displayName} left ${member.guild.name}`);
+        const repository = AppDataSource.getRepository(UserRank);
+        const userRank = await removeMembers(member, repository);
+        if (userRank) repository.save(userRank);
+    }
+
+    @DiscordEvent("guildDelete")
+    async guildDelete(guild: Guild) {
```
Comment (TotallyNot) [inline src/handler/job/monitorRanks.ts:78]: or this?

File: src/handler/job/monitorRanks.ts (hunk 1 above)
Comment (Natty-Boh) [reply src/handler/job/monitorRanks.ts]: this is outdated, already fixed. was an accident

File: src/handler/job/monitorRanks.ts (hunk 2 above)
Comment (Natty-Boh) [reply src/handler/job/monitorRanks.ts:70]: it should be fine, it wont remove the guild id for the user, but that doesn't really matter.

it'll just mean that the application will waste time trying to fetch guilds the user isnt in.

File: src/handler/job/monitorRanks.ts (hunk 3 above)
Comment (Natty-Boh) [reply src/handler/job/monitorRanks.ts:78]: same as above

File: src/handler/job/monitorRanks.ts (hunk 2 above)
Comment (TotallyNot) [reply src/handler/job/monitorRanks.ts:70]: ok, we can just add a cleanup job later that removes dead users then

**Review by TotallyNot — APPROVED** [review]

**Pull request merged**